content/starter-kits/create-wc-dapp.md

```diff
@@ -0,0 +1,15 @@
+---
+title: 'Create WalletConnect Dapp'
+description: 'Bootstrap a WalletConnect integrated dapp with in seconds 📲'
+authors: ['@WalletConnect', '@Boidushya']
+tags: ['Dapp', 'Web3', 'WalletConnect', 'Web3Modal']
```

Comment:
Can you please remove project specific tags? Thanks :)

Reply:
> Can you please remove project specific tags? Thanks :)

Removed 🫡